crates/biome_grit_formatter/tests/spec_tests.rs

```diff
@@ -3,6 +3,10 @@ mod spec_test;
 mod formatter {
 
     mod grit_module {
-        tests_macros::gen_tests! {"tests/specs/grit/**/*.grit", crate::spec_test::run, ""}
+        tests_macros::gen_tests! {"tests/specs/grit/*.grit", crate::spec_test::run, ""}
```

Comment:
What is the intention of this file change?

Reply:
Ah, I broke it a part to better debug the individual tests. I was using the VSCode extension to run the debugger, and for whatever reason it wouldn't show the individual test that's specific to the `where` pattern`. I can move these back together though instead of being separate modules.

Reply:
Thank you for explaining the reason behind this change, but I suggest reverting the changes to this file because we need to ensure that all tests in the `tests/specs/grit` folder are covered. However, after adding these changes, some tests will not be covered. For example, if we add a new test in `tests/specs/grit/predicate/foo.grit` in the future, this test file will not be covered.

Reply:
Makes sense! Reverting.